internal/parser/simple_parser_rules.go

```diff
@@ -51,23 +51,37 @@ func (p *simpleParser) parseSQLStatement(r reporter) (stmt *ast.SQLStmt) {
 	case token.KeywordCommit:
 		stmt.CommitStmt = p.parseCommitStmt(r)
 	case token.KeywordCreate:
-		p.parseCreateStmt(stmt, r)
+		p.parseCreateStmts(stmt, r)
 	case token.KeywordDelete:
-		stmt.DeleteStmt = p.parseDeleteStmt(r)
+		p.parseDeleteStmts(stmt, nil, r)
 	case token.KeywordDetach:
 		stmt.DetachStmt = p.parseDetachDatabaseStmt(r)
+	case token.KeywordDrop:
+		p.parseDropStmts(stmt, r)
```

Comment:
Why is there nothing assigned?

Reply:
Takes in stmt pointer, and assigns inside.